### PR TITLE
feature/281

### LINKED ISSUE #281: Replace terrain.svg with original work
We need to create a little symbol to represent terrain to replace the existing icon, which is the only file in the entire repo that is under a different license than MIT.

### PR DESCRIPTION
**Pull request by RNGFavorsNoobs**
Resolves #281.

Updated `terrain.svg` and `README.md`.

Corrected simple grammatical errors in README.md as well.

### CODE REVIEW

File: assets/images/icons/terrain.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="36" height="36" viewBox="0 0 35.999999 35.999999"><path d="M27 23H9l.893-1.55 2.78-4.816c.493-.851 1.295-.851 1.786 0 0 0 .645 1.118.723 1.249l2.34-4.055c.494-.85 1.296-.85 1.787 0l3.915 6.783c.202-.35.401-.694.553-.964.357-.614.934-.614 1.292 0 .354.617.934 1.622 1.29 2.234L27 23z" fill="#fff"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 122.3 84.1"><defs><style>.cls-1{fill:#fff;}</style></defs><title>Asset 1</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M0,84.1,37.9,11.3,50.7,36.1,72.3,0l50,83.8Z"/></g></g></svg>
```
Comment (n8rzz) [inline assets/images/icons/terrain.svg]: the viewBox on yours is significantly different than the original.

Comment (n8rzz) [inline assets/images/icons/terrain.svg]: here is what I'm talking about on the sizing:

screen shot 2017-03-05 at 10 19 45 pm

Comment (RNGFavorsNoobs) [inline assets/images/icons/terrain.svg]: Ah, will look into it.

**Comment by n8rzz** [comment]
@erikquinn I'd like to get your sign off on this one before we hit the button.  Looks good to me.

**Comment by n8rzz** [comment]
@erikquinn bump

**Comment by erikquinn** [comment]
:shipit: with v4.3.0 please.

**Comment by erikquinn** [comment]
Changes to readme reverted, as they are covered by #74.

**Pull request merged**